conda-recipe/meta.yaml

```diff
@@ -19,13 +19,13 @@ requirements:
         - setuptools
         - cython
         - numba 0.54*
-        - dpctl >=0.9*
+        - dpctl >=0.9.1.dev*
```

Comment:
This will presently make it break, since no such package is yet available on the channels.

Reply:
@oleksandr-pavlyk 
If API changed then dpctl version should be 0.10.0dev0.
0.9.1 is a patch version.
Right?

Reply:
To play nice with multi-versioning on Linux - yes.